pkg/pillar/Dockerfile

```diff
@@ -92,6 +92,12 @@ ENV GOARCH=${TARGETARCH}
 ENV CC=${CROSS_COMPILE_ENV}gcc
 ARG GOPKGVERSION
 
+# Temporary gomodule patches until accepted upstream
```

Comment:
We've never patched go modules directly like this... IMO would be better to push this modified version to a fork on GH (you did these changes, right? So you can push them to a forked repo under your account) and use the fork as the GO module (changing at the go.mod). Once the changes become upstream, you can update the go.mod to fetch from the official repo again...

Reply:
Yes I authored this patch.  Sounds good, I'll make the changes to importing a module instead.

Reply:
Done